packages/test/test-end-to-end-tests/src/test/pendingBatchReentry.spec.ts

```diff
@@ -170,9 +170,7 @@ describeCompat(
 
 				test.initial();
 				containerRuntime.orderSequentially(() => {
-					containerRuntime.ensureNoDataModelChanges(() => {
-						test.reentrant();
-					});
+					test.reentrant();
```

Comment:
I have a feeling the change to this test is no longer doing reentrancy. Reentrancy was an issue about creating ops while processing an op (as reflected by the change made to `ContainerRuntime.process(...)`).
All this test is doing now is submitting an op wrapped in `orderSequentially`, which won't inherently be reentrant.

Reply:
To be honest, I do not understand what it tests. I'll it as is, as API continues to exist on ContainerRuntime (not on interface), and this code reaches into ContainerRuntime class.